install/kubernetes/helm/istio/charts/galley/templates/deployment.yaml

```diff
@@ -66,6 +66,7 @@ spec:
           - --enable-validation=false
 {{- end }}
           - --validation-webhook-config-file
+          - --enable-reconcileWebhookConfiguration={{ .Values.enableReconcileWebhookConfiguration }}
```

Comment:
Please move it before `- --validation-webhook-config-file`, it's breaking `validation-webhook-config-file` flag.

Reply:
Thanks! Done.